vendor/directx/dxgi/dxgi.odin

```diff
@@ -117,10 +118,20 @@ MWA_VALID :: MWA{
 	.NO_PRINT_SCREEN,
 }
 
+SHARED_RESOURCE_RW :: distinct bit_set[SHARED_RESOURCE_RW_FLAG; u32]
+SHARED_RESOURCE_RW_FLAG :: enum u32 {
+    READ  = 31,
```

Comment:
Please use tabs to indent codebase. I recommend installing `.editorconfig` extension available for various editors that should configure your editor to use the correct indentation automatically.